ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/FooCommandGenerator.scala

```diff
@@ -30,21 +34,25 @@ final class FooCommandGenerator(
       .toMap
   private val observersWithUnlikelihood: List[(Primitive.Party, Int)] =
     allObservers.zipWithIndex.toMap.view.mapValues(unlikelihood).toList
+  private val divulgeesWithUnlikelihood: List[(Primitive.Party, Int)] =
```

Comment:
`Unlikelihood` is an unusual construct.  Would weight or probability be better ?

Reply:
This is an inverse of the probabality: the bigger the value, the less likely it is it'll be chosen.
Does `improbablity`sound better?

Reply:
Although that may please some hitchhikers I'm not sure it would help the average developer.  The probability is 1/10^n ?

Reply:
> The probability is 1/10^n ?

Correct

Reply:
I may _use an integer in the construction of the probability_ and then pass the probability around (i.e. not the Int).